projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,86 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+* Installed VDK, Control Service, Jupyter instance, vdk-jupyterlab-extension, vdk-notebook, vdk-ipython
```

Comment:
Include links to dictionary , the first time you introduce a term - https://github.com/vmware/versatile-data-kit/wiki/dictionary

Or link to the plugin page (for plugins)